### PR TITLE
[BPK-4324] improve docs components paths

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -13,6 +13,17 @@
   },
   "settings": {
     "import/resolver": {
+     "alias" : {
+        "map": [
+          ["helpers",  "./docs/src/helpers" ],
+          ["components", "./docs/src/components"],
+          ["static",  "docs/src/static" ],
+          ["backpack",  "./backpack" ],
```
Comment (olliecurtis) [inline .eslintrc:21]: I thought we don't have this repo included in Backpack anymore as we use the npm packages directly?

File: babel.config.js [hunk 2]
```diff
@@ -24,6 +24,21 @@ module.exports = function config(api) {
...
       '@babel/plugin-proposal-class-properties',
+      [
+        'module-resolver',
+        {
+          root: ['./'],
+          alias: {
+            helpers: './docs/src/helpers',
+            components: './docs/src/components',
+            static: './docs/src/static',
+            backpack: './backpack',
```
Comment (olliecurtis) [inline babel.config.js:35]: Same here

File: jsconfig.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                "./docs/src//helpers"
+            ],
+            "components": [
+                "./docs/src/components"
+            ],
+            "static": [
+                "docs/src/static"
+            ],
+            "backpack": [
+                "backpack"
```
Comment (olliecurtis) [inline jsconfig.json:15]: Same here

**Review by olliecurtis — CHANGES_REQUESTED** [review]

**Review by olliecurtis — COMMENTED** [review]
Just tried the PR build and seems the ios page is having issues with the screenshots

Screenshot 2020-12-14 at 16 44 56

**Comment by georgegillams** [comment]
> Just tried the PR build and seems the ios page is having issues with the screenshots
> 
> Screenshot 2020-12-14 at 16 44 56

It can't load the android ones either, and I didn't yet change their path.
So either I've broken everything, or the PR deploy process isn't perfect yet

**Comment by olliecurtis** [comment]
Yeah just saw that too! 

Tried it on this url too and its the same https://backpack.github.io/prs/854/components/text?platform=native

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**